src/ibm/CMakeLists.txt

```
@@ -0,0 +1,37 @@
+set(IBM_INCLUDE_DIR ${CMAKE_SOURCE_DIR}/include/ibm)
+
+find_package(jansson REQUIRED)
+
+add_library(backend_ibm SHARED)
+target_sources(backend_ibm PRIVATE ibm.c)
+target_include_directories(backend_ibm PUBLIC ${IBM_INCLUDE_DIR} ${QDMI_INCLUDE_DIRS} ${QDMI_SRC_DIRS} ${QINFO_INCLUDE_DIRS})
+target_link_libraries(backend_ibm PUBLIC qdmi jansson)
```

```suggestion
find_package(jansson REQUIRED)

add_library(backend_ibm SHARED ibm.c)
target_include_directories(backend_ibm PUBLIC ${CMAKE_SOURCE_DIR}/include/ibm ${QDMI_SRC_DIRS})
target_link_libraries(backend_ibm PUBLIC qdmi jansson)
```
This should be enough for now.
In the future, we should not be relying on internal header details and the `${QDMI_SRC_DIRS}` should be gone as well.

Please also remove the commented out code and add a new line at the end of the file.

Please apply a similar reasoning to the other CMake files.